vignettes/irecist.Rmd

```diff
@@ -562,8 +562,7 @@ adrs <- adrs %>%
       PARCAT3 = "iRECIST",
       AVAL = yn_to_numeric(AVALC),
       ANL01FL = "Y"
-    ),
-    check_type = "none"
```

Comment:
I think this line shouldn't be removed because `irsp_y` and `icb_y` could select the same record.